src/svg/ds6/icons.svg

```diff
@@ -223,4 +223,13 @@
     <symbol id="icon-window" viewBox="0 0 28 25">
         <path d="M.497 5.701h15.644c.274 0 .497.222.497.497v.718a.497.497 0 0 1-.497.497H.497A.497.497 0 0 1 0 6.916v-.718c0-.274.222-.497.497-.497zm1.215 0V23.94H0V5.701h1.712zm18.515 7.785v9.939a.552.552 0 0 1-.552.552h-.699a.552.552 0 0 1-.552-.552v-9.939c0-.305.247-.552.552-.552h.699c.305 0 .552.247.552.552zM0 22.302h20.209v1.712H0v-1.712zM25.767.161h1.914v16.583h-1.914V.161zm1.914-.147v1.914H11.098V.014h16.583zM26.322 0l1.353 1.353L13.398 15.63l-1.353-1.353L26.322 0z"/>
     </symbol>
+	<symbol id="icon-confirmation" viewBox="0 0 14 14">
+		<path fill="#5ba71b" fill-rule="evenodd" d="M7 14A7 7 0 1 0 7 0a7 7 0 0 0 0 14zM3.976 6.68a.553.553 0 0 0-.809 0 .621.621 0 0 0 0 .848l2.285 2.296a.553.553 0 0 0 .808 0l4.573-4.8a.621.621 0 0 0 0-.848.553.553 0 0 0-.809 0L5.856 8.55 3.976 6.68z"/></defs>   
+	</symbol>
+	<symbol id="icon-notice" viewBox="0 0 14 14">
+		<path fill="#3665f3" d="M7 0C3.136 0 0 3.136 0 7s3.136 7 7 7 7-3.136 7-7-3.136-7-7-7zm0 11.94c.7.017.7-.781.7-.781V5.734S7.7 4.989 7 5c-.7.011-.7.734-.7.734v5.425s0 .763.7.78zM7 3.8a.7.7 0 1 0 0-1.4.7.7 0 0 0 0 1.4z"/>
+	</symbol>
+	<symbol id="icon-error" viewBox="0 0 14 14">
+		<path fill="#e62048" fill-rule="evenodd" d="M7 0C3.136 0 0 3.136 0 7s3.136 7 7 7 7-3.136 7-7-3.136-7-7-7zm0 9.34c.7.017.7-.781.7-.781V3.134s0-.745-.7-.734c-.7.011-.7.734-.7.734v5.425s0 .763.7.78zm0 2.46a.7.7 0 1 0 0-1.4.7.7 0 0 0 0 1.4z"/>
```

Comment:
If you remove `fill-rule`, does it behave correctly?

Reply:
Checking on this now

Reply:
Update: If I remove `fill-rule` it won't recognize the "inner" icons (`!`, `i`, and checkmark)

Reply:
Thought so. Thanks for checking!